packages/xo-web/src/xo-app/xoa/licenses/index.js

```diff
@@ -6,31 +6,87 @@ import Link from 'link'
 import React from 'react'
 import renderXoItem from 'render-xo-item'
 import SortedTable from 'sorted-table'
-import {
-  addSubscriptions,
-  adminOnly,
-  connectStore,
-  getXoaPlan,
-  ShortDate,
-} from 'utils'
+import { addSubscriptions, adminOnly, connectStore, ShortDate } from 'utils'
+import { CURRENT } from 'xoa-plans'
 import { Container, Row, Col } from 'grid'
 import { createSelector, createGetObjectsOfType } from 'selectors'
-import { find, forEach } from 'lodash'
 import { get } from '@xen-orchestra/defined'
-import { subscribePlugins, getLicenses } from 'xo'
+import {
+  subscribePlugins,
+  getLicenses,
+  selfBindLicense,
+  subscribeCurrentLicense,
+} from 'xo'
+import { find, forEach, some } from 'lodash'
 
 import Xosan from './xosan'
 
-const openNewLicense = () => {
-  // FIXME: use link with target attribute
-  window.open('https://xen-orchestra.com/#!/member/purchaser')
-}
+// -----------------------------------------------------------------------------
+
+const LicenseManager = ({ item, userData }) => {
+  const { type } = item
+
+  if (type === 'xosan') {
+    const { srId } = item
+
+    if (srId === undefined) {
+      return _('licenseNotBoundXosan')
+    }
+
+    const sr = userData.xosanSrs[srId]
+    if (sr === undefined) {
+      return _('licenseBoundUnknownXosan')
+    }
+
+    return <Link to={`srs/${sr.id}`}>{renderXoItem(sr)}</Link>
```

Comment:
You can use `render-xo-item/Sr` component.

Reply:
Unrelated to this PR, I'll do it later.